src/service/rating.tsx

```diff
@@ -0,0 +1,49 @@
+const ratingDataInLocalStorage: { storageKey: string; rating: Array<{}> } = {
+  storageKey: 'movie-rating-data',
+  rating: []
+};
+
+const ratingData = {
+  getMovieRatingFromStorage() {
+    const movieRatingStorage: any = localStorage.getItem(
```

Comment:
Maybe it is string?

Reply:
Thanks for a hint. I have changed a type to string, then a got a message "type of null" for movieRatingStorage is not acceptable to a 'type string', after that a changed a type to a string | null and now it seems to work